### PR TITLE
refactor: missing accessories system

### PR DESCRIPTION
**Pull request by DuckySoLucky**
## Description

Uses `/items` endpoint to get accessories instead of hard coding every one of them
cc @MattTheCuber

### CODE REVIEW

File: src/lib.js [hunk 1]
```diff
@@ -1147,8 +1147,8 @@ export const getItems = async (
       }
     }
 
-    if (id in constants.ACCESSORY_UPGRADES) {
-      const accessoryUpgrades = constants.ACCESSORY_UPGRADES[id];
+    if (constants.getUpgradeList(id) !== undefined) {
```
Comment (MattTheCuber) [inline src/lib.js]: `!== undefined` is redundant (used several places). Unless you want to exclude other falsy values, but I don't think you do

Comment (DuckySoLucky) [reply src/lib.js:1150]: > `!== undefined` is redundant (used several places). Unless you want to exclude other falsy values, but I don't think you do

Just realized it returns boolean, I'm dumb ;-;

**Comment by DuckySoLucky** [comment]
Also fixes inactive accessories not being shown
> Before

image

> After

image

**Comment by DuckySoLucky** [comment]
Found bug, cba to fix it today

**Comment by DuckySoLucky** [comment]
> Found bug, cba to fix it today

Fixed

**Comment by DuckySoLucky** [comment]
> Also fixes inactive accessories not being shown

Made it better, now it shows accessories which aren't in accessory bag or aren't unique as well
> Before

image
> After

image

**Comment by DuckySoLucky** [comment]
Added `Location` field to inactive accessories so players can see where the accessory is located at
image

**Comment by MattTheCuber** [comment]
> Made it better, now it shows accessories which aren't in accessory bag or aren't unique as well

What about accessories you don'teet requirements for like Reaper Orb?

**Comment by DuckySoLucky** [comment]
> > Made it better, now it shows accessories which aren't in accessory bag or aren't unique as well
> 
> What about accessories you don'teet requirements for like Reaper Orb?

~~Good idea, gonna add that. Most likely tomorrow~~
Edit: Nothing about that, SkyCrypt doesn't like that for some reason..

**Review by MartinNemi03 — APPROVED** [review]
lgtm, tested on only a few profiles

**Pull request merged**